From 6a419585f2be2167e0a8c71e214936acbd11ce9a Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Thu, 26 Jul 2018 15:41:50 +0300
Subject: [PATCH 0819/1200] mvebu_mcbin-88f8040_defconfig: enable
 CONFIG_DISPLAY_BOARDINFO

This allows to print information regarding the operating
frequencies of the board and SoC information, for
example:
	Model: Marvell Armada 8040 MACCHIATOBin
	SoC: Armada8040-A2; AP806-A1; 2xCP110-A2
	Clock:  CPU     1300 [MHz]
	        DDR     800  [MHz]
	        FABRIC  800  [MHz]
	        MSS     200  [MHz]

Change-Id: I087e3016770d7f671654dfa009e14c082bd10c7a
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/58449
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 configs/mvebu_mcbin-88f8040_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/mvebu_mcbin-88f8040_defconfig b/configs/mvebu_mcbin-88f8040_defconfig
index 719d1bd224..01a2ee1d21 100644
--- a/configs/mvebu_mcbin-88f8040_defconfig
+++ b/configs/mvebu_mcbin-88f8040_defconfig
@@ -14,7 +14,6 @@ CONFIG_BOOTCOMMAND="run get_images; run set_bootargs; booti $kernel_addr $ramfs_
 CONFIG_SYS_CONSOLE_ENV_OVERWRITE=y
 CONFIG_SYS_CONSOLE_INFO_QUIET=y
 # CONFIG_DISPLAY_CPUINFO is not set
-# CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_ARCH_EARLY_INIT_R=y
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_BOARD_EARLY_INIT_F=y
-- 
2.22.0

